librz/bin/format/coff/coff_reloc.c

```diff
@@ -35,6 +35,43 @@ RZ_API ut64 rz_coff_import_index_addr(struct rz_bin_coff_obj *obj, ut64 imp_inde
 
 typedef void (*RelocsForeachCb)(RZ_BORROW RzBinReloc *reloc, ut8 *patch_buf, size_t patch_buf_sz, void *user);
 
+static size_t reloc_general_arch_rel32_common(struct rz_bin_coff_obj *bin, struct coff_reloc *rel, RzBinReloc *reloc, ut64 sym_vaddr, ut8 *patch_buf) {
```

Comment:
what about adding `name` as argument which is used to set `print_name`, so you don't have to specify name afterwards?